src/CameraVapix.ts

```diff
@@ -212,166 +201,73 @@ export class CameraVapix extends EventEmitter {
         return eventName == 'eventsConnect' || eventName == 'eventsDisconnect';
     }
 
-    eventsConnect(channel = 'RTSP') {
+    eventsConnect(): void {
         if (this.ws != null) {
             throw new Error('Websocket is already opened.');
         }
-        if (this.rtsp != null) {
-            throw new Error('RTSP is already opened.');
-        }
-        if (channel == 'RTSP') {
-            this.rtspConnect();
-        } else if (channel == 'websocket') {
-            this.websocketConnect();
-        } else {
-            throw new Error('Unknown channel.');
-        }
-    }
-
-    eventsDisconnect() {
-        if (this.rtsp != null) {
-            this.rtsp.disconnect();
-        }
-        if (this.ws != null) {
-            this.ws.close();
-        }
-    }
-
-    private rtspConnect() {
-        this.rtsp = new RtspClient({
+        const options: WsClientOptions = {
+            tls: this.tls,
+            tlsInsecure: this.tlsInsecure,
+            auth: this.auth,
             ip: this.ip,
             port: this.port,
-            auth: this.auth,
-        });
+            address: '/vapix/ws-data-stream?sources=events',
+        };
+        this.ws = new WsClient(options);
```

Comment:
ws je jineho typu nez tam prirazujes..to si TS nestezuje? ws bys v tomhle souboru asi  nepotreboval ani importovat..

Reply:
V jiných souborech si stěžoval.
!